Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -224,6 +224,10 @@ - (instancetype)initWithWeakPtr:(WeakPtr<WebKit::WebProcessPool>&&)weakPtr
 namespace WebKit {
 using namespace WebCore;
 
+#if PLATFORM(MAC)
+static RetainPtr<NSApplication> protectedNSApp() { return NSApp; }
+#endif
```

Comment:
I’d like @rniwa or @geoffreygaren to comment on this idiom.

Reply:
Removed since rniwa commented checker should know NSApp is a constant